src/Factory/MemcachedFactory.php

```diff
@@ -28,7 +29,7 @@ class MemcachedFactory extends AbstractAdapterFactory
      */
     public function getAdapter(array $config)
     {
-        $client = new \Memcached();
+        $client = new Memcached();
```

Comment:
Looks good to me :)

On 20 Jul 2016, at 17:15, Tobias Nyholm <notifications@github.com> wrote:

> In src/Factory/MemcachedFactory.php:
> 
> > @@ -28,7 +29,7 @@ class MemcachedFactory extends AbstractAdapterFactory
> >       */
> >      public function getAdapter(array $config)
> >      {
> > -        $client = new \Memcached();
> > +        $client = new Memcached();
> Sorry, My bad.
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub, or mute the thread.
>